pkg/services/ngalert/api/tooling/Makefile

```diff
@@ -25,6 +29,30 @@ spec.json-mac: ensure_go-swagger_mac $(GO_PKG_FILES)
 post.json: spec.json
 	go run cmd/clean-swagger/main.go -if $(<) -of $@
 
-.PHONY: openapi
-openapi: post.json
+swagger-codegen-api:
+	docker run --rm -v $$(pwd):/local --user $$(id -u):$$(id -g) swaggerapi/swagger-codegen-cli generate \
+	-i /local/post.json \
+	-l go-server \
+	-Dapis \
+	-o /local \
+	--additional-properties packageName=api \
+	-t /local/swagger-codegen/templates \
+	# --import-mappings eval.RelativeTimeRange="github.com/grafana/grafana/pkg/services/ngalert/eval" \
+	# --type-mappings RelativeTimeRange=eval.RelativeTimeRange
+
+copy-files:
+	ls -1 go | xargs -n 1 -I {} mv go/{} ../generated_base_{}
+
+fix:
+	sed -i -e 's/apimodels\.\[\]PostableAlert/apimodels.PostableAlerts/' $(GENERATED_GO_MATCHERS)
+	sed -i -e 's/apimodels\.\[\]UpdateDashboardAclCommand/apimodels.Permissions/' $(GENERATED_GO_MATCHERS)
+	sed -i -e 's/apimodels\.\[\]PostableApiReceiver/apimodels.TestReceiversConfigParams/' $(GENERATED_GO_MATCHERS)
+	goimports -w -v $(GENERATED_GO_MATCHERS)
+
+clean:
+	rm -rf ./go
+
+serve: post.json
```

Comment:
Previously, the tooling would start serving a [Swagger UI](https://swagger.io/tools/swagger-ui/) instance on port 80 as the last step. The makefile would always need to be Ctrl+C'd.

Instead, don't launch this by default. Put this functionality under `make serve` instead.